### ISSUE TITLE
[20.05] Identification Parameters: No process found in process group

### ISSUE THREAD

**Issue by CarlosHorro** [galaxyproject/galaxy#9935]
Hi,

I'm doing some testing with the Identification Parameters tool in the Test ToolShed, using for the dependency my own conda repo 'carloshorro' (installation process works perfectly).
This tool hasn't had any change for a long time and it's quite simple (no mandatory input), but now trying Galaxy 20.05 I found that it crashes with a no very descriptive error:

```
galaxy.jobs.runners.util.process_groups DEBUG 2020-06-29 11:38:22,598 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] check_pg(): No process found in process group 42226
galaxy.jobs.runners.local DEBUG 2020-06-29 11:38:22,599 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] execution finished: /home/carlos/GitHub/galaxy/database/jobs_directory/000/24/galaxy_24.sh
galaxy.tool_util.output_checker INFO 2020-06-29 11:38:22,619 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] Job 24: {'type': 'exit_code', 'desc': 'Fatal error: Exit code 127 (Job Failed)', 'exit_code': 127, 'code_desc': 'Job Failed', 'error_level': 3}
galaxy.tool_util.output_checker DEBUG 2020-06-29 11:38:22,620 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] Tool exit code indicates an error, failing job.
galaxy.tool_util.output_checker DEBUG 2020-06-29 11:38:22,620 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] job failed, detected state generic_error, standard error is - []
galaxy.jobs DEBUG 2020-06-29 11:38:22,723 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] (24) setting dataset 39 state to ERROR
galaxy.jobs INFO 2020-06-29 11:38:23,001 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] Collecting metrics for Job 24 in /home/carlos/GitHub/galaxy/database/jobs_directory/000/24
galaxy.tools.error_reports DEBUG 2020-06-29 11:38:23,049 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] Bug report plugin <galaxy.tools.error_reports.plugins.sentry.SentryPlugin object at 0x7fcc5d64a040> generated response None
galaxy.jobs DEBUG 2020-06-29 11:38:23,083 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] job_wrapper.finish for job 24 executed (450.087 ms)
```

The only thing it may be indicative is this "check_pg(): No process found in process group " line.
Mention that I was able to execute the same tool in some Galaxy 19 instance...

Greetings,
Carlos

**Comment by mvdbeek** [comment]
> galaxy.tool_util.output_checker INFO 2020-06-29 11:38:22,619 [p:30727,w:1,m:0] [LocalRunner.work_thread-2] Job 24: {'type': 'exit_code', 'desc': 'Fatal error: Exit code 127 (Job Failed)', 'exit_code': 127, 'code_desc': 'Job Failed', 'error_level': 3}

Exit code 127 means your dependency could not be found. You can check the dependency status in the `manage dependencies` panel  in the admin menu.

**Comment by CarlosHorro** [comment]
Yep, I checked it and you were right. The problem had its source in that I was keeping the same version number even although I updated the Conda requirement. It was a curious case, because it was trying to use a Conda package already existing in my system although the specified dependency was a different one. Increasing the version number of the tool solved the problem.

Thank you very much,
Carlos
